From c1058f5acf6aa79d6a47e1dbb7dc1c4df93cd31b Mon Sep 17 00:00:00 2001
From: Dominique Martinet <asmadeus@codewreck.org>
Date: Fri, 14 Jun 2024 22:32:43 +0900
Subject: [PATCH 4/4] kernel headers: don't print warning without -v

---
 src/utils.cpp | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/utils.cpp b/src/utils.cpp
index c358a401fc83..6b16c45e3a7e 100644
--- a/src/utils.cpp
+++ b/src/utils.cpp
@@ -743,7 +743,9 @@ std::tuple<std::string, std::string> get_kernel_dirs(
   auto has_ksrc = is_dir(ksrc);
   auto has_kobj = is_dir(kobj);
   if (!has_ksrc && !has_kobj) {
-    LOG(WARNING) << "Could not find kernel headers in " << ksrc << " or "
+    // will never work on nixos, print as debug instead until we figure something out
+    if (bt_verbose)
+    LOG(DEBUG) << "Could not find kernel headers in " << ksrc << " or "
                  << kobj
                  << ". To specify a particular path to kernel headers, set the "
                     "env variables BPFTRACE_KERNEL_SOURCE and, optionally, "
-- 
2.44.0

