diff --git a/qt/aqt/main.py b/qt/aqt/main.py
index 990398c53f7c..ed65066e56d5 100644
--- a/qt/aqt/main.py
+++ b/qt/aqt/main.py
@@ -1201,6 +1201,9 @@ def updateTitleBar(self) -> None:
     ##########################################################################
 
     def setupAutoUpdate(self) -> None:
+        # Don't check for latest version since the versions are
+        # managed in Nixpkgs.
+        return
         import aqt.update
 
         self.autoUpdate = aqt.update.LatestVersionFinder(self)
